src/main/cpp/geo/Profile.cpp

```diff
@@ -0,0 +1,38 @@
+#include <fstream>
+#include <boost/tokenizer.hpp>
+#include "Profile.h"
+#include "util/StringUtils.h"
+
+namespace stride {
+namespace geo {
+
+ProfileRef Profile::Parse(std::ifstream& csv_file)
+{
+	std::vector<City> cities;
```

Comment:
The brace style is okay, but `std::vector<City>` was indented differently from the other statements in this file. Seems fixed now. :+1: